packages/perseus-editor/src/components/locked-figure-select.tsx

```diff
@@ -40,6 +40,13 @@ const LockedFigureSelect = (props: Props) => {
                     >
                         Line
                     </ActionItem>,
+                    <ActionItem
+                        key={`${id}-vector`}
+                        label="Vector"
+                        onClick={() => onChange("vector")}
+                    >
+                        Vector
+                    </ActionItem>,
```

Comment:
No need to do anything, just an FYI: In my upcoming Locked Circle PR, I'm changing this to a map.